cg/server/admin.py

```diff
@@ -323,7 +324,17 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
+        "invoiced_at",
```

Comment:
I would like to keep the column for the Invoice (ie the column that is currently the last one on the right). I use it to keep track of what samples have been invoiced. I am not sure what this column is called, could be invoiced_at but I am not sure.

Reply:
You are probably thinking about the "Invoice" column, and we'll keep it visible